algorithms/Python/README.md

```diff
@@ -26,6 +26,7 @@
 - [n-th Fibonacci number](recursion/nth_fibonacci_number.py)
 - [Recursive Insertion Sort](recursion/recursive_insertion_sort.py)
 - [Recursive Sum of n numbers](recursion/recursive-sum-of-n-numbers.py)
+- [Depth First Search](recursion/depth-first-search.py)
```

Comment:
I think a better place for dfs should be under graph section rather than under recursion section

Reply:
Sorry I didn't think about it before. I will change change that. Thank you for pointing that out.